### PR TITLE
Add PTG splash screen on hippodrome

### PR DESCRIPTION
**Pull request by alabobriggs**
The previous splash screen wasn't clear that you can get PTG from hippodrome. 

Screen Shot 2021-08-25 at 09 21 53

### CODE REVIEW

File: packages/hippodrome-ui/src/components/SplashScreens.tsx [hunk 1]
```diff
@@ -52,6 +54,49 @@ const SplashScreen: React.FC<SplashScreenProps> = () => {
     },
   ]
 
+  if (ptgSwap) {
```
Comment (tobowers) [inline packages/hippodrome-ui/src/components/SplashScreens.tsx:57]: seems like this should be its own component?

File: packages/hippodrome-ui/src/layouts/Layout.tsx [hunk 2]
```diff
@@ -106,14 +95,20 @@ const Layout: React.FC = ({ children }) => {
         <CurrentUser />
       </HStack>
 
-      {!address && <SplashScreen />}
+      {!address && <SplashScreen ptgSwap={location.pathname === "/ptg"} />}
```
Comment (tobowers) [inline packages/hippodrome-ui/src/layouts/Layout.tsx:98]: this should be a route, right?

File: packages/hippodrome-ui/src/components/SplashScreens.tsx [hunk 3]
```diff
@@ -52,6 +54,53 @@ const SplashScreen: React.FC<SplashScreenProps> = () => {
...
+            <Heading
+              as="h2"
+              fontFamily="zed-dots"
+              fontSize="48px"
+              fontWeight="normal"
+              textAlign="center"
+            >
+              Swap{" "}
+              <Box as="span" color="brandOrange.500">
+                BTC
```
Comment (tobowers) [inline packages/hippodrome-ui/src/components/SplashScreens.tsx]: "Bitcoin"

**Review by tobowers — COMMENTED** [review]
couple of questions on the routing and we're working through the colors in discord.

File: packages/hippodrome-ui/src/layouts/Layout.tsx (hunk 2 above)
Comment (alabobriggs) [reply packages/hippodrome-ui/src/layouts/Layout.tsx:98]: yep

**Comment by alabobriggs** [comment]
@tobowers updated

File: packages/hippodrome-ui/src/layouts/Layout.tsx [hunk 4]
```diff
@@ -106,14 +96,28 @@ const Layout: React.FC = ({ children }) => {
         <CurrentUser />
       </HStack>
 
-      {!address && <SplashScreen />}
+      {!address && (
+        <>
+          {location.pathname === "/ptg" ? (
```
Comment (tobowers) [inline packages/hippodrome-ui/src/layouts/Layout.tsx:101]: shouldn't this be a route?

**Review by tobowers — COMMENTED** [review]
just same routing comment as before

File: packages/hippodrome-ui/src/layouts/Layout.tsx (hunk 4 above)
Comment (alabobriggs) [reply packages/hippodrome-ui/src/layouts/Layout.tsx:101]: I don't get the question

Comment (tobowers) [reply packages/hippodrome-ui/src/layouts/Layout.tsx:101]: sorted out in discord

**Review by tobowers — APPROVED** [review]
good first step!

**Pull request merged**